components/BugZap/BugZapGame.js

```diff
@@ -91,15 +92,18 @@ class BugZapGame extends React.Component {
         this.givenTime = this.props.route.givenTime - 500;
       }
       if (this.trialNumber > LEVEL2_TRIALS) {
+        // now blackout and spotlight shown before trials
+        this.getSpotLightStyle();
         this.blackout = true;
       }
     } else {
       // first trial, run through all animations once
       this.setCharacterAnimations();
     }
 
-
-    this.setCharacterDirection();
+    if (this.directionMaySwitch) {
```

Comment:
I chose "may" because this is only saying that the character is allowed to switch now, we don't know if it will switch sides, that is determined by another function